src/style.css

```
@@ -0,0 +1,177 @@
+html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 198px;
```

effectively you can rid of a pack of margins if you define the card paddings
and be more specific with sizing if used `box-sizing: border-box;`:
```suggestion
  box-sizing: border-box;
  width: 200px;
  padding: 32px 16px 16px;
```